assets/src/dashboard/app/views/myStories/content/storyListItem/components.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { Icons } from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import {
+  STORY_PREVIEW_WIDTH,
+  VIEWPORT_BREAKPOINT,
+} from '../../../../../constants';
+import { TableRow, MoreVerticalButton } from '../../../../../components';
+
+export const PreviewImage = styled.div`
+  display: inline-block;
+  background: ${({ theme }) => theme.colors.gradient.placeholder};
+  width: ${STORY_PREVIEW_WIDTH[VIEWPORT_BREAKPOINT.THUMBNAIL]}px;
+  height: ${STORY_PREVIEW_WIDTH[VIEWPORT_BREAKPOINT.THUMBNAIL] / (3 / 4)}px;
+  object-fit: contain;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+`;
+export const PreviewWrapper = styled.div`
+  position: relative;
+`;
+
+export const LockIcon = styled(Icons.LockClosed)`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 24px;
+  height: 24px;
```

Comment:
not sure if this matters too much, but I'd either do:
```
  top: 0;
  left: 0;
  width: 24px;
  height: 24px;
```
or
```
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
```

both is kind of confusing. do you want this to be 24x24? or take up the full size of its next relative anscestor?

Reply:
🤔  so, why i did this was so absolutely center the lock icon on top of the poster image and make the svg 24px. 
I know i could split this up and use a div for the icon but it seemed redundant since this works. Thoughts? 
 
| Top/ Right + Width/Height | Top/Right/Left/Bottom | Top/Left/Right/Bottom + Height |
| --- | --- | --- |
| <img width="364" alt="Screen Shot 2021-07-28 at 8 57 20 AM" src="https://user-images.githubusercontent.com/10720454/127356316-cc5cf968-e800-47f0-b757-34136ef87ceb.png"> | <img width="373" alt="Screen Shot 2021-07-28 at 8 57 39 AM" src="https://user-images.githubusercontent.com/10720454/127356331-fd4f06c1-ab94-43cd-9481-71d643f76f25.png"> | <img width="370" alt="Screen Shot 2021-07-28 at 8 57 47 AM" src="https://user-images.githubusercontent.com/10720454/127356323-7704ea66-6e0b-4c91-a5bc-5b5e7ea5be41.png"> |

Reply:
I mean, I guess if it works, it works. so it's a fine solution.

I guess I'm surprised it works just because it feels conflicting. if it's centered, it's left and right aren't really `0` same with the top and bottom. but 🤷 . I have other solutions that I'd normally reach for, and happy to share, but I guess they're not needed since this works fine

Reply:
found a compromise :) 
```
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
```

Does the same thing, easier to read.

Reply:
yea that solution makes sense